hash_practice/exercises.py

```diff
@@ -5,17 +5,56 @@ def grouped_anagrams(strings):
         Time Complexity: ?
         Space Complexity: ?
     """

```

Comment:
Calling out the missing complexity calculations here, but I haven't been dinging people for it.